### PR TITLE
Add base style file for consumption via web components

### PR DESCRIPTION
**Pull request by jshcrowthe**
# PR Summary
@lambertsteven this is the PR we discussed about getting Web Component support in the styleguide.

## Pieces:
- Add base.styl file that is compiled to dist
- Add `$limit-base-styles` flag to remove classes from typography in the base styles
- Add `@font-face` definintions to the fonts section of `_variables.styl`
- Add `fs-styles.html` file to the root of the project and to `bower.json`
- Rev version to `v1.8`

### CODE REVIEW

File: assets/css/elements/_typography.styl [hunk 1]
```diff
@@ -88,63 +88,121 @@ body {
...
  */
-h1, h2, h3, h4,
-.fs-h1, .fs-h2, .fs-h3, .fs-h4 {
-  color: $color-grey;
-  font-family: $font-face-heading;
-  letter-spacing: inherit;
-  text-transform: inherit;
-  margin-top: 0;
-}
+if $limit-base-styles
```
Comment (lambertsteven) [inline assets/css/elements/_typography.styl]: are the classes so much of a bad thing in the base styles that the duplication in code is less problematic?

Comment (jshcrowthe) [inline assets/css/elements/_typography.styl]: It's just to ensure that if someone *happens* (however unlikely) to use that exact class name in the component to do an override that the base styles.

**Comment by lambertsteven** [comment]
I'll pull if you remove the conditional variable (to remove code duplication in the stylus file) and remove the `@font-face` rules. If we provide the font-face rules in the styles, we allow 3rd party users to use our fonts off our CDN, and our fonts are specifically licensed for our use. We'll have to add the font-face rules into theme-engage so only our site uses them.

**Comment by jshcrowthe** [comment]
Alright, I will go through and remove the large duplication block and see if I can make that smaller. I do think that having the conditional there is valuable. As to the fonts is that just something that our components are enhanced by being on our site (and therefore having the font available) and other places they just default to the fallback?

**Comment by lambertsteven** [comment]
Ya, the components would be enhanced by being on our site.

**Comment by jshcrowthe** [comment]
@lambertsteven I have updated the code. I still stand by the conditionals being there but I agree with your point that we shouldn't have to edit code in two places. I have updated the `_typography.styl` to use mixins for each style grouping.

**Pull request merged**